HIP/hip-830.md

```diff
@@ -0,0 +1,350 @@
+---
+hip: 830
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2023-12-04T07:00:00Z
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: 2023-12-04
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the `birthRound` to the metadata of new events. The `birthRound` value is 1 + the latest round to have come
+   to consensus at the time the event was created. This value is used to determine if the event is ancient or not.  
+   It is also used to look up the correct consensus roster for validating the event's signature.
+2. Represent references to parent events as `EventDescriptors`. EventDescriptors combine the existing data about each
+   parent (hash, creator, generation) and the birthRound into a single data structure.
+3. Allow for multiple other parents.
+4. In Gossip, reduce an event's unhashed metadata to just the event's signature.
```

Comment:
the "In Gossip" part here will be confusing for anyone not intimately familiar with the event structure in the code. I suggest:

```suggestion
4. Reduce `EventUnhashedData` to just the event's signature. This reduction eliminates two deprecated sequence number fields and the other event parent ID field which is already stored in in the other parent `EventDescriptor`.
```